client/cli/src/params.rs

```diff
@@ -578,74 +594,12 @@ pub struct RunCmd {
 	pub password_filename: Option<PathBuf>
 }
 
-/// Stores all required Cli values for a keyring test account.
-struct KeyringTestAccountCliValues {
-	help: String,
-	conflicts_with: Vec<String>,
-	name: String,
-	variant: sp_keyring::Sr25519Keyring,
-}
-
-lazy_static::lazy_static! {
-	/// The Cli values for all test accounts.
-	static ref TEST_ACCOUNTS_CLI_VALUES: Vec<KeyringTestAccountCliValues> = {
-		sp_keyring::Sr25519Keyring::iter().map(|a| {
-			let help = format!(
-				"Shortcut for `--name {} --validator` with session keys for `{}` added to keystore.",
-				a,
-				a,
-			);
-			let conflicts_with = sp_keyring::Sr25519Keyring::iter()
-				.filter(|b| a != *b)
-				.map(|b| b.to_string().to_lowercase())
-				.chain(std::iter::once("name".to_string()))
-				.collect::<Vec<_>>();
-			let name = a.to_string().to_lowercase();
-
-			KeyringTestAccountCliValues {
-				help,
-				conflicts_with,
-				name,
-				variant: a,
-			}
-		}).collect()
-	};
-}
-
 /// Wrapper for exposing the keyring test accounts into the Cli.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, StructOpt)]
 pub struct Keyring {
 	pub account: Option<sp_keyring::Sr25519Keyring>,
 }
 
-impl StructOpt for Keyring {
```

Comment:
Hmm, this is another change I don't really want to see. `--alice` is much shorter than `--account alice`

Reply:
Oops, that was a mistake. I will change that